apps/src/templates/progress/styles.scss

```diff
@@ -1,6 +1,24 @@
 @import 'color';
```

Comment:
Could you comment a bit on why you chose styles.scss for this file?  I have a couple of observations/concerns:
1. We don't seem to have similarly named files elsewhere in the tree?
2. When importing styles, we import it as 'styles' in some files and as 'css' in other files (presumably because styles is already taken).  Maybe this suggests that the name 'styles' is too broad?

Reply:
great observations. turns out that was a typo. we have a bunch of `style.scss` files in the repo so it seemed like a reasonably choice, but i misspelled it! our convention is to name the js-defined styles objects we use in most components `styles` (probably what lead to my typo), which is why in some components i had to choose something other than `styles`, hence `css`.

updating the name to `style.scss` would address both of your concerns (it would match an existing pattern, and i could always import it as `style` therefore avoiding the need for the alternate `css` naming). that said, i'm not married to this name. i was expecting it might be something we'd want to discuss. 

do you have any better suggestions? `progressStyles` is taken, and `progressTableStyles` no longer applies... however, i could restore `progressTableStyles` and rename this to `bubbleStyles`... for some reason when i moved and renamed this i felt an aversion to creating a separate .scss file just for two little bubble-specific style definitions, but looking at it now i think that might actually be the best option. thoughts?

Reply:
`bubbleStyles` sounds good to me!